crates/pop-contracts/src/test.rs

```diff
@@ -46,7 +46,7 @@ mod tests {
 		crate::create_smart_contract(
 			"test_contract",
 			temp_contract_dir.as_path(),
-			&crate::Template::Standard,
+			&crate::ContractTemplate::Standard,
```

Comment:
Do we need a borrow here?

Reply:
Yes, `create_smart_contract` requires a borrowed template.